src/app/services/defects/defects.service.ts

```diff
@@ -0,0 +1,20 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Defect } from '@models/defects/defect.model';
+import { Observable } from 'rxjs';
+import { environment } from '../../../environments/environment';
+
+@Injectable({ providedIn: 'root' })
+export class DefectsService {
+  private url = `${environment.VTM_API_URI}/defects/`;
```

Comment:
I think it would be best practice to not have the trailing slash. Add necessary slash to added segments e.g. `/id`

Reply:
Because of that trailing slash, line 18 was nice and clean. But I do agree with the convention, so suggestion applied.